packages/ods/src/components/tabs/src/components/ods-tabs/ods-tabs.scss

```diff
@@ -5,7 +5,15 @@
 
   display: inline-flex;
   flex-flow: row;
-  padding: var(--ods-outline-width);
-  width: inherit;
-  overflow-x: auto;
+  padding: var(--ods-outline-width) 0;
+  width: 100%;
+  overflow-y: auto;
+}
+
+.ods-tabs {
+  &-border {
```

Comment:
- use `__` for BEM block
- use a more generic name, as this block contains the tabs, it's not just for the border (something like `__items` or `__wrapper` or `__container`)